test/release/examples/primers/errorHandling.chpl

```diff
@@ -11,7 +11,10 @@
   ---------------
   Several important standard library functions throw errors so that developers
   can handle the errors as they see fit. This is especially useful for
-  recovering from errors in :mod:`IO`.
+  recovering from errors in :mod:`IO`. The `try` and `catch` constructs
+  enable handling of errors. Any errors caught by a `catch` block are `owned`
+  errors and can be rethrown. `try!` is available to halt if an error is
```

Comment:
I think we prefer to use two ` to wrap code instead of just one.  In the generated docs, what you currently have gets italicized instead of made into an inline code block